### ISSUE TITLE
Please update dependancies

### ISSUE THREAD

**Issue by diegosa891** [atn832/fake_cloud_firestore#273]
C:\flutter\bin\flutter.bat --no-color pub upgrade
Resolving dependencies...
Because every version of fake_cloud_firestore depends on cloud_firestore_platform_interface ^5.0.1 and cloud_firestore >=4.10.0 depends on cloud_firestore_platform_interface ^6.0.0, fake_cloud_firestore is incompatible with cloud_firestore >=4.10.0.
So, because adroit_air depends on both cloud_firestore ^4.10.0 and fake_cloud_firestore ^2.4.1+1, version solving failed.
Process finished with exit code 1

**Comment by tsinis** [comment]
Agree, it blocks most of `firebase_*` packages :(

**Comment by atn832** [comment]
Here's a quick workaround until I get around to publishing an update. In your pubspec.yaml, you can add a dependency override:
```
dependency_overrides:
  cloud_firestore_platform_interface: ^6.0.0
```
https://dart.dev/tools/pub/dependencies#dependency-overrides

**Comment by atn832** [comment]
Thanks for reporting the issue. Just upgraded the dependencies and published version 2.4.2. https://pub.dev/packages/fake_cloud_firestore/changelog#242
